enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -1302,18 +1305,13 @@ func nonCmdExit(ctx context.Context, err error) *interfaces.CommandResult {
 func (c *FirecrackerContainer) newID(ctx context.Context) error {
 	vmIdxMu.Lock()
 	defer vmIdxMu.Unlock()
-	u1, err := uuid.NewRandom()
-	if err != nil {
-		return err
-	}
-	u2, err := uuid.NewRandom()
-	if err != nil {
-		return err
-	}
+	uuid1 := uuid.New()
+	uuid2 := uuid.New()
+
 	vmIdx += 1
-	log.CtxDebugf(ctx, "Container id changing from %q (%d) to %q (%d)", c.id, c.vmIdx, u1.String(), vmIdx)
-	c.id = u1.String()
-	c.snapshotID = u2.String()
+	log.CtxDebugf(ctx, "Container id changing from %q (%d) to %q (%d)", c.id, c.vmIdx, uuid1, vmIdx)
+	c.id = uuid1
+	c.snapshotID = uuid2
```

Comment:
nit: revert?

Reply:
I did this because I use the util/uuid package elsewhere and it felt silly to have both imported. Agree it's a little noisy though.

Reply:
ah woops I misinterpreted the diff here - lgtm